trin-storage/src/versioned/id_indexed_v1/store.rs

```diff
@@ -85,13 +89,14 @@ impl VersionedContentStore for IdIndexedV1Store {
             pruning_strategy,
             usage_stats: UsageStats::default(),
             metrics: StorageMetricsReporter::new(protocol_id),
+            phantom_content_key: PhantomData,
```

Comment:
Do you mean you've seen that in many places other than this codebase? I saw that `_` prefix being used a fair amount in other places, but inside the codebase we've dropped the `_` prefix, so I just followed suit for consistency. But now it seems (based off your comment and what I've seen online) that the `_` prefix is convention, so I'll go ahead and make all the changes here

Reply:
Yeah, in other casebases, as a general convention. I wasn't aware that we dropped it in our codebase.
I'm fine with whatever you decide (using `_` because of the general convention, or leaving it as it is for consistency within codebase).